vaas/vaas/settings/base.py

```diff
@@ -78,6 +74,10 @@ def serialize(value: any) -> str:
     'simple_history.middleware.HistoryRequestMiddleware',
 ]
 
+# Plugins definition
+INSTALLED_APPS += tuple(env.json('INSTALLED_PLUGINS', default=[]))
```

Comment:
I'm afraid that You unintentionally changed important logic. INSTALLED_PLUGINS deliberately has precedence before other INSTALLED_APPS.